terraform/environments/core-network-services/firewall-rules/preproduction_rules.json

```diff
@@ -160,6 +167,13 @@
     "destination_port": "443",
     "protocol": "TCP"
   },
+  "hmpps_preproduction_to_pgres_tcp": {
+    "action": "PASS",
+    "source_ip": "${hmpps-preproduction}",
+    "destination_ip": "0.0.0.0/0",
```

Comment:
Can you change this one to reference `${cloud-platform}` for the destination please?

Reply:
addressed